sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/SeleniumITHelper.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.test.aad.common;
+
+import com.azure.spring.test.AppRunner;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.UUID;
+import java.util.regex.Pattern;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SeleniumITHelper {
+
+    protected AppRunner app;
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    private final static String uuid = UUID.randomUUID().toString();
+
+    static {
+        init();
+        deleteChromeDriverFile();
```

Comment:
This method name could be confusing, we should change it to something like `registerShutdownHookxxxxx`